app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt

```diff
@@ -88,7 +88,7 @@ class SuggestedEditsImageTagsFragment : SuggestedEditsItemFragment(), CompoundBu
 
         binding.tagsLicenseText.text = StringUtil.fromHtml(getString(R.string.suggested_edits_cc0_notice,
                 getString(R.string.terms_of_use_url), getString(R.string.cc_0_url)))
-        binding.tagsLicenseText.movementMethod = LinkMovementMethodExt.getInstance()
```

Comment:
What's the reason for this change? It doesn't seem related to Kotlin conversion.

Reply:
Use `LinkMovementMethodExt.getInstance()` will show a "Unresolved reference: getInstance". 

Since this looks like a general usage of  `LinkMovementMethod` and it should be fine to use `LinkMovementMethod.getInstance()`.